src/Controller/MenuController.java

```diff
@@ -162,6 +167,23 @@
 	private GridPane mainContent;
 	@FXML
 	private HBox topBox;
+	//chat variables
+	private final BorderPane mainPane = new BorderPane();
+	private final GridPane firstPane = new GridPane();
+	private final GridPane userMessagePane = new GridPane();
+	private final HBox spacingBox = new HBox();
+	private TextField tfName = new TextField("");
+	private TextField tfHost = new TextField("");
+	private TextField tfMessageToSend = new TextField();
+	private TextArea msgArea = new TextArea();
+	private final Label name = new Label("Name:");
+	private final Label host = new Label("Host:");
+	private final Button submitButton = new Button("Submit");
+	private final Button sendButton = new Button("Send");
+
+	private String suserName;
+	private String shostName;
+	private int nportNumber = 1111;
```

Comment:
What is the purpose of this variable?  It appears that it is never used.  Also, if it is kept, it looks like it should be called `nPortNumber` (notice the second letter capitalized).

Reply:
I was going to implement it into my GUI but decided for future purposes to leave it.  When I work on the peer to peer chat I wanted to already have it in here that way I can just get a little bit of a head start.  I can remove it if you would like till my next pull request